app/javascript/controllers/sidebar_group_controller.js

```diff
@@ -1,10 +1,16 @@
+/* global IntersectionObserver */
+
 import { Controller } from '@hotwired/stimulus'
 
 export default class extends Controller {
   static targets = ['title', 'list', 'link']
 
   connect () {
     this.toggleShow()
+    if (this.isEditOrganization()) {
+      this.toggleShowAnchorList()
+      this.menuHighlight()
+    }
```

Comment:
I think it may make more sense to do this in the html. There is a bootstrap class `collapse` that controls whether the menu is open or closed. It may make more sense to modify the component a bit so that you can pass that in and use rails to check whether that should be open or not via some helper.

Reply:
Maybe... I'm second guessing myself because I am not 100% on how easy it would be to set that initial state not in javascript.

Reply:
If it works let's keep it for now :)